lib/child_process.js

```diff
@@ -111,7 +109,7 @@ exports.fork = function fork(modulePath /* , args, options */) {
 };
```

Comment:
Can we get rid of these unnecessary semicolons now that the assignments are gone?

Reply:
sure, i will do later

Reply:
thanks, done